mxcube3/routes/Queue.py

```diff
@@ -532,6 +532,40 @@ def get_default_char_params():
     resp.status_code = 200
     return resp
 
+@mxcube.route("/mxcube/api/v0.1/queue/mesh", methods=['GET'])
+def get_default_mesh_params():
+    """
+    returns the default values for a mesh.
+    """
+    acq_parameters = mxcube.beamline.get_default_acquisition_parameters('default_mesh_values')
+    resp = jsonify({
+        'acq_parameters': {
+            'first_image': acq_parameters.first_image,
+            'num_images': acq_parameters.num_images,
+            'osc_start': acq_parameters.osc_start,
+            'osc_range': acq_parameters.osc_range,
+            'kappa': acq_parameters.kappa,
+            'kappa_phi': acq_parameters.kappa_phi,
+            'overlap': acq_parameters.overlap,
+            'exp_time': acq_parameters.exp_time,
+            'num_passes': acq_parameters.num_passes,
+            'resolution': acq_parameters.resolution,
+            'energy': acq_parameters.energy,
+            'transmission': acq_parameters.transmission,
+            'shutterless': acq_parameters.shutterless,
+            'detector_mode': acq_parameters.detector_mode,
+            'inverse_beam': False,
+            'take_dark_current': True,
+            'skip_existing_images': False,
+            'take_snapshots': True,
+            'cell_counting': mxcube.beamline['default_mesh_values'].getProperty('cell_counting', 'None'),
```

Comment:
Maybe replace None with a method that we know exists, like zig-zag so that we by accident won't get None as method ?